orangecontrib/educational/widgets/owkmeans.py

```diff
@@ -135,85 +137,71 @@ class OWKmeans(OWWidget):
     # other settings
     k_means = None
     autoPlaySpeed = settings.Setting(1)
-    lines_to_centroids = settings.Setting(0)
+    lines_to_centroids = settings.Setting(True)
     graph_name = 'scatter'
     outputName = "cluster"
-    button_labels = {"step1": "Reassign membership",
-                     "step2": "Recompute centroids",
-                     "step_back": "Step back",
-                     "autoplay_run": "Run",
-                     "autoplay_stop": "Stop",
-                     "random_centroids": "Randomize"}
+    STEP_BUTTONS = ["Reassign Membership", "Recompute Centroids"]
+    AUTOPLAY_BUTTONS = ["Run", "Stop"]
     colors = ['#2f7ed8', '#0d233a', '#8bbc21', '#910000', '#1aadce',
               '#492970', '#f28f43', '#77a1e5', '#c42525', '#a6c96a']
 
     def __init__(self):
         super().__init__()
 
         # options box
-        self.optionsBox = gui.widgetBox(self.controlArea)
-        self.cbx = gui.comboBox(self.optionsBox, self, 'attr_x',
-                                label='X:',
-                                orientation=Qt.Horizontal,
-                                callback=self.restart,
-                                sendSelectedValue=True)
-        self.cbx.setSizePolicy(QSizePolicy(QSizePolicy.MinimumExpanding, QSizePolicy.Fixed))
-        self.cby = gui.comboBox(self.optionsBox, self, 'attr_y',
-                                label='Y:',
-                                orientation='horizontal',
-                                callback=self.restart,
-                                sendSelectedValue=True)
-        self.cby.setSizePolicy(QSizePolicy(QSizePolicy.MinimumExpanding, QSizePolicy.Fixed))
+        self.optionsBox = gui.widgetBox(self.controlArea, "Data")
+        opts = dict(
+            widget=self.optionsBox, master=self, orientation=Qt.Horizontal,
+            callback=self.restart, sendSelectedValue=True)
```

Comment:
It's totally subjective, but when multiple controls share some settings, I prefer to put them in one dict, so it guards the consistency and lets me modify all controls in one place.